src/app/TargetSelect/TargetSelect.tsx

```diff
@@ -251,11 +263,24 @@ export const TargetSelect: React.FunctionComponent<TargetSelectProps> = (props)
 
   return (
     <>
-      <Card>
-        <CardHeader>
-          <CardHeaderMain>
+      <Card isExpanded={props.simple || selected === NO_TARGET ? undefined : isExpanded}>
```

Comment:
looks like `props.simple || selected === NO_TARGET` being used a lot. Maybe, we should extract into a React memo?